src/lib.rs

```diff
@@ -1,6 +1,6 @@
 use std::{
-    env, fs,
-    fs::File,
+    env,
+    fs::{self, File},
```

Comment:
Personally I'm really against importing `self` and then a few other items into scope, rather then also qualifying them with e.g. `fs::File` if I really wanted to import `fs` into scope.

Reply:
Do you have some logical reasoning behind that? I don't really care about that stuff personally.